test/parallel/test-http-malformed-request.js

```diff
@@ -25,11 +25,11 @@ const assert = require('assert');
 const net = require('net');
 const http = require('http');
 const url = require('url');
+const Countdown = require('../common/countdown');
 
 // Make sure no exceptions are thrown when receiving malformed HTTP
 // requests.
-
-let nrequests_completed = 0;
+const nrequests_completed = 0
```

Comment:
Can you try to run `make lint` or `make test` after you have made changes? I think this might give you an error because usually semicolons are required by the linter?

Reply:
I ran `make lint` found the `new line` error and an absent semicolon